src/Cipher.ts

```diff
@@ -305,37 +289,19 @@ export function createDecipheriv(
 export function createCipher(
   algorithm: string,
   password: BinaryLike,
-  options?: Stream.TransformOptions
+  options?: Record<string, unknown> | Stream.TransformOptions
```

Comment:
and here

Reply:
https://github.com/margelo/react-native-quick-crypto/pull/245/files#r1548175650